### PR TITLE
Add automodsumm_included_members option, take2

### PR DESCRIPTION
**Pull request by bsipocz**
This is the rebased version of #103 (as that one was opened from an org fork, maintainers cannot directly push to it)

EDIT: close #121

### CODE REVIEW

**Comment by og113** [comment]
Hi all, really appreciate the library you've made. This feature would be really useful to us, and it looks pretty much ready. Could someone possibly take a look and see if it can be merged please? Thank you :)

File: sphinx_automodapi/automodapi.py [hunk 1]
```diff
@@ -93,6 +93,12 @@
     Should be a bool and if ``True`` members that a class inherits from a base
     class are included in the generated documentation. Defaults to ``False``.
 
+* ``automodsumm_included_members``
+    A list of strings containing the names of hidden class members that should be
+    included in the documentation. This is most commonly used to add special class
+    methods like ``__getitem__`` and ``__setitem__``. Defaults to
+    ``['__init__', '__call__']``.
+
```
Comment (saimn) [inline sphinx_automodapi/automodapi.py:101]: It's a bit confusing to document this option in the `automodapi` module. Some options are documented in both modules, some other are not ...

**Comment by pllim** [comment]
This didn't make it to 0.16.0 . Looks like there is an unresolved comment?

**Comment by bsipocz** [comment]
Rebased and I would not simmer this along much more, what about rolling a YOLO on it @pllim?

**Review by pllim — APPROVED** [review]
Sure. Let's try it out.

**Comment by pllim** [comment]
Thanks, all!

**Pull request merged**